tracing-journald/src/lib.rs

```diff
@@ -268,6 +298,60 @@ where
     }
 }
 
+/// The facility argument is used to specify what type of program is logging the
+/// message. This lets the configuration file specify that messages from
+/// different facilities will be handled differently.
+#[derive(Copy, Clone)]
+pub enum SyslogFacility {
+    /// security/authorization messages
+    Auth,
+    /// security/authorization messages (private)
+    Authpriv,
+    /// clock daemon (cron and at)
+    Cron,
+    /// system daemons without separate facility value
+    Daemon,
+    /// ftp daemon
+    Ftp,
+    /// kernel messages (these can't be generated from user
+    /// processes)
+    Kern,
+    ///reserved for local use
+    Local0,
+    ///reserved for local use
+    Local1,
+    ///reserved for local use
+    Local2,
+    ///reserved for local use
+    Local3,
+    ///reserved for local use
+    Local4,
+    ///reserved for local use
+    Local5,
+    ///reserved for local use
+    Local6,
+    ///reserved for local use
+    Local7,
+    /// line printer subsystem
+    Lpr,
+    /// mail subsystem
+    Mail,
+    /// USENET news subsystem
+    News,
+    /// messages generated internally by syslogd(8)
+    Syslog,
+    /// generic user-level messages
+    User,
+    /// UUCP subsystem
```

Comment:
nit: generally, the documentation comments for enum variants are capitalized:
```suggestion
    /// Security/authorization messages
    Auth,
    /// Security/authorization messages (private)
    Authpriv,
    /// Clock daemon (`cron` and `at`)
    Cron,
    /// System daemons without separate facility value
    Daemon,
    /// FTP daemon
    Ftp,
    /// Kernel messages (these can't be generated from user
    /// processes)
    Kern,
    /// Reserved for local use
    Local0,
    /// Reserved for local use
    Local1,
    /// Reserved for local use
    Local2,
    /// Reserved for local use
    Local3,
    /// Reserved for local use
    Local4,
    /// Reserved for local use
    Local5,
    /// Reserved for local use
    Local6,
    /// Reserved for local use
    Local7,
    /// Line printer subsystem
    Lpr,
    /// Mail subsystem
    Mail,
    /// USENET news subsystem
    News,
    /// Messages generated internally by `syslogd(8)`
    Syslog,
    /// Generic user-level messages
    User,
    /// UUCP subsystem
```